pygmt/datasets/samples.py

```diff
@@ -80,6 +81,7 @@ def load_sample_data(name):
 
     # Dictionary of private load functions
     load_func = {
+        "compositions": _load_sample_compositions,
```

Comment:
Are the compositions for a rock? If yes, I would prefer a name like `rock_compositions` and a function name `_load_rock_compositions`.

Reply:
I think this is a good name for the dataset.

Reply:
Sounds good.